README.md

```diff
@@ -5,9 +5,9 @@ The official Ruby library for integrating with the Onfido API.
 [![Gem Version](https://badge.fury.io/rb/onfido.svg)](http://badge.fury.io/rb/onfido)
 [![Build Status](https://travis-ci.org/onfido/onfido-ruby.svg?branch=master)](https://travis-ci.org/onfido/onfido-ruby)
 
-This version of the gem uses `v3.1` of the Onfido API and is compatible with Ruby 2.4 onwards.
+Documentation can be found at https://documentation.onfido.com
```

Comment:
@Phoebe-B @superdevx64 We used to have a way of linking to the docs so the Ruby code examples would already be shown (something like `documentation.onfido.com/?language=ruby`) - does that still exist?

Reply:
@Phoebe-B May be worth creating a ticket for this, it was a pretty good feature and useful for linking here, but up to you